portfolio/src/main/webapp/style.css

```diff
@@ -1,20 +1,17 @@
-/* #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-} */
 
 body{
```

Comment:
You should add a space between the selector and the opening brace.

Take a look at some css style guides:
https://google.github.io/styleguide/htmlcssguide.html#CSS
https://github.com/airbnb/css